src/documents/APIChangesFrom2_0To3_0.md

```diff
@@ -17,10 +17,17 @@ This section describes major APIs that were in common use. For information about
 
 #### Panel Changes ####
 
+"Links and Buttons" adjuster and enactors collapsed to a single preference, "Enhance Links"
```

Comment:
"... and enactors are collapsed... ", missing "are". Also, might be better to replace the comma with a dash or a colon?